### DISCUSSION TITLE
Filtering by environment not working as expected

### DISCUSSION THREAD

**Discussion by codeflow-karma** [getsentry/support#59] [Support]
In my Sentry UI, I am able to select production, staging or all (by deselecting both). But the numbers dont add up. I get a certain count of issues when the filter is set to production, but when I set it to all (by deselecting both), I get to see more issues, that are for the environment production. I've opened up one such issue and am looking at it from all angles to see what causes this issue to not appear when the filter is set to production environment. Any clues, anyone?

**Comment by Kobby-Bawuah** [comment]
Hello,

This seems off, is this instance self-hosted? Do you mind writing into support to make it a bit easier to debug the issue?
